notebooks/community/vertex_endpoints/optimized_tensorflow_runtime/bert_optimized_online_prediction.ipynb

```diff
@@ -1510,7 +1510,7 @@
       "source": [
```

Comment:
<p>Replace "For details, ..."  with</p><p> </p><p>Learn more about [Classify text with BERT](&nbsp;<a href="https://www.tensorflow.org/text/tutorials/classify_text_with_bert" target="_blank">https://www.tensorflow.org/text/tutorials/classify_text_with_bert</a>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/630/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>